charm.go

```diff
@@ -99,3 +101,52 @@ func IsUnsupportedSeriesError(err error) bool {
 	_, ok := err.(*unsupportedSeriesError)
 	return ok
 }
+
+// MaybeCreateVersionFile creates/overwrite charm version file.
+func MaybeCreateVersionFile(path string) error {
+	var charmVersion string
```

Comment:
Don't declare charmVersion here - it is used way down below.
Variables need to be defined/initialised in the block where they're used.